### ISSUE TITLE
withRequestOptions behaviour for multi-follow API traversal

### ISSUE THREAD

**Issue by paulwilton** [basti1302/traverson#25]
Hi
When invoking a multi-follow API traversal, with templated URLs, 
eg
```javascript
.follow("my:path1", "my:path2")
.withTemplateParameters([null, null, { 'myParam': value }])
```
we are able to force the template to be applied to a specific request in the request chain (in this case myParam is templated only against my:path2

However, this doesn't seem to be possible when using withRequestOptions() in multi follow API traversal.
eg 
```javascript
.follow("my:path1", "my:path2")
.withRequestOptions({ 'qs' : { 'myParam': value }})
```
applies the request options to each follow request, ie to the root API request, and to both my:path1 and my:path2.

Is it possible to make withRequestOptions work in the same manner as withTemplateParameters ?

thanks, Paul.

**Comment by basti1302** [comment]
Currently, this is not possible. I'll leave this open as a feature request.

I'm currently knee deep in some other projects. This is why work on Traverson has come to a halt for the last 2-3 months. I'll start working on Traverson again soon (~2 weeks) and will have a look at it then. Sorry for the delay.

**Comment by basti1302** [comment]
This has landed in 1.1.0
